.travis.yml

```diff
@@ -49,6 +49,13 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]] && [[ "$INSTALL_QEMU_ARM" == "yes" ]]; then tools/apt-get-install-qemu-arm.sh; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then tools/brew-install-deps.sh; fi
   - if [[ "$OPTS" == "--test262" ]]; then sudo timedatectl set-timezone America/Los_Angeles; fi
+  - if [[ "$OPTS" == *"--check-pylint"* ]];
+    then
+      git clone https://github.com/PyCQA/pylint.git -b pylint-1.6.5 "$PWD/third-party/pylint";
```

Comment:
What if we do the following? ` pip install pylint==1.6.5`
Then yo don't need to manually call clone/setup.